content/docs/04-ai-sdk-rsc/04-generative-ui.mdx

```diff
@@ -1,11 +1,25 @@
 ---
 title: Generative User Interfaces
-description: Overview of the AI and UI states
+description: Overview of generative user interfaces
 ---
 
 # Generative User Interfaces
 
-Since language models can render user interfaces as part of their generations, the resulting model generations are referred to as generative user interfaces.
+Generative user interfaces elevate language models beyond plaintext and markdown. By rendering dynamic, adaptive user interfaces, language models can proactively adapt to the needs of the user.
+
+Built for use with React Server Components, `ai/rsc` is optimized for delivering highly-dynamic, personalized software like an e-commerce store, a web app, a social network -- and the most dynamic interfaces of them all: generative user interfaces. The benefits of using React Server Components for generative user interfaces include:
+
+- Reduced client bundle sizes
+- Fewer round trips between the server and the client
+- Improved developer experience around managing backend and frontend code
+
+<div className="mb-4">
+  <Note>
+    {' '}
```

Comment:
question for reviewers: we should add a CTA here, where can we point users to for this callout?

Reply:
good idea!

Reply:
jeremy is working on a full fledged example for this! will merge when ready, reverting this PR to draft